tests/blockchain/regtest/SimpleTransfer.cpp

```diff
@@ -88,18 +68,20 @@ TEST_F(Regtest_fixture_simple, DISABLED_send_to_client)
          number_of_test++) {
 
         SendCoins(*receiver, *sender, target_height, coin_to_send);
-        std::this_thread::sleep_for(std::chrono::seconds(20));
```

Comment:
What changed from previous versions that we do not need those sleeps ?

Reply:
The same what in previous sleep